packages/tupaia-web/src/types/mapOverlay.d.ts

```diff
@@ -1,33 +0,0 @@
-/*
```

Comment:
Great to be able to delete this

Reply:
Do we need to delete the line from types/index.d.ts as well?

Reply:
Good catch, removed